index.bs

```diff
@@ -466,15 +521,13 @@ spec: webrtc-pc; urlPrefix: https://www.w3.org/TR/webrtc/#
           return a Promise rejected with a <code>TypeError</code>.
         </li>
         <li>
-          If <code>configuration.video</code> in non-null and
-          <code>configuration.video.contentType</code> is not a <a>valid
-          VideoConfiguration contentType</a>, return a Promise rejected with a
+          If <code>configuration.video</code> is <a>present</a> and is not a
```

Comment:
Re: support bit, fine to push this out, but I have a nuanced change to previous proposal.

I think foolip's suggestion to just treat invalid is not supported for nonsensical things like 'video/webm; codecs="foo"', but I do think we should still throw an exception for strings that are simply not formatted correctly like

'video/webm garbag codecs=vp8"'

Note the lack of an opening quote around vp8.